lib/oauth-db.js

```diff
@@ -1,35 +1,55 @@
 var Boom = require('boom');
 var crypto = require('crypto');
 var Hoek = require('hoek');
+var pg = require('pg');
+var queries = require('./queries');
+var connString = process.env.POSTGRE_CONNECTION_STRING;
 
-var OAuthDB = function OAuthDB(clients, authorizations, tokens) {
-  this.clients = {};
-  this.authCodes = {};
-  this.accessTokens = {};
+var salt = process.env.TOKEN_SALT;
 
-  clients.forEach(function(client) {
-    this.clients[client.client_id] = client;
-  }, this);
+Hoek.assert(salt, 'You must define a TOKEN_SALT for salting auth codes and access tokens');
 
-  Object.keys(authorizations).forEach(function(auth) {
-    this.authCodes[auth] = authorizations[auth];
-  }, this);
-
-  tokens.forEach(function(token) {
-    this.accessTokens[token.access_token] = token;
-  }, this);
+var OAuthDB = function OAuthDB(Adapter) {
```

Comment:
Is this Adapter parameter being used for something? Maybe just needs to be removed?